### ISSUE TITLE
ENH: repeat gene legend every say 30 samples in case of long list

### ISSUE THREAD

**Issue by corneliusroemer** [lenaschimmel/sc2rf#18]
when I have 100 samples, the legend with gene boundaries goes out of screen. It would be nice if the legend could be repeated.

**Comment by lenaschimmel** [comment]
Then this would make sense for the whole header block with coordinates, genes, ref, primers... right? Or if you are just interested in gene boundaries, and someone else just in primers, etc. then it should be configurable what is repeated and what isn't.

**Comment by corneliusroemer** [comment]
Yeah it's getting a bit murky here to keep this all in a terminal application...

In a spreadsheet, you'd pin row 1 to always have it at the top, but not sure the complexity is worth it, now that I think about it.
